package.json

```diff
@@ -43,7 +43,8 @@
     "prepublishOnly": "npm run build",
     "preversion": "npm run lint",
     "start": "concurrently --kill-others -n Server,Typescript,Rollup 'browser-sync start -s -f build examples --serveStatic examples' 'tsc -w' 'rollup -c -w'",
-    "test": "jest"
+    "test": "jest",
+    "watch:test": "jest --watch"
```

Comment:
Just fyi, you can also run `yarn jest --watch`.

Reply:
thx, didn't know it. Still, it is a good thing to have for those who don't know it -- when they do an `npm run`, they will see it as an option (in case they didn't know about it, or if they don't want to look through jest docs)